packages/experimental-utils/src/ts-eslint/RuleTester.ts

```diff
@@ -46,7 +46,12 @@ interface RunTests<
   invalid: InvalidTestCase<TMessageIds, TOptions>[];
 }
 interface RuleTesterConfig {
-  parser: '@typescript-eslint/parser';
+  parser?:
```

Comment:
These are the ones listed here: https://eslint.org/docs/user-guide/configuring#specifying-parser

added `string` as well in case there are others